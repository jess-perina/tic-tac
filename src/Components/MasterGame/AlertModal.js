import React from 'react'

function AlertModal(props) {
  return (
    <div>
      <a className="modal-trigger" href="#!" />
      <div id="modal1" className="modal">
        <div className="modal-content">
          <h5>reset current game to change options</h5>
        </div>
        <div className="modal-footer">
          <a href="#!" className="modal-close waves-effect waves-green btn-flat">CANCEL</a>
          <a href="#!" className="modal-close waves-effect waves-green btn-flat" onClick={props.reset} >RESET</a >
        </div>
      </div>
    </div>
  )
}

export default AlertModal
