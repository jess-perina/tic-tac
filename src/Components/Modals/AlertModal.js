import React from 'react'

const AlertModal = ({ reset }) => {
  return (
    <div>
      <div id="modal1" className="modal">
        <div className="modal-content">
          <h5>reset current game to change options</h5>
        </div>
        <div className="modal-footer">
          <a href="#!" className="modal-close waves-effect waves-green btn-flat">CANCEL</a>
          <a href="#!" className="modal-close waves-effect waves-green btn-flat" onClick={reset} >RESET</a>
        </div>
      </div>

      <div id="modal2" className="modal">
        <div className="modal-content">
          <h5>Game Rules</h5>
        </div>
        <div className="modal-footer">
          <a href="#!" className="modal-close waves-effect waves-green btn-flat">OK</a>
        </div>
      </div>
    </div>
  )
}

export default AlertModal
