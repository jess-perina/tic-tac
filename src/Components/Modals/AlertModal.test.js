import React from 'react'
import { shallow } from 'enzyme'
import AlertModal from './AlertModal'

describe('AlertModal Component', () => {
  it('renders as expected', () => {
    expect(shallow(<AlertModal />)).toMatchSnapshot()
  })
})