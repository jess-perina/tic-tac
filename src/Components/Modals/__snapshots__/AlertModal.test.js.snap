// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertModal Component renders as expected 1`] = `ShallowWrapper {}`;
